src/test/java/core/basesyntax/service/impl/ReportGenerationImplTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportGeneration;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReportGenerationImplTest {
+    private Map<Fruit,Integer> map;
+    private ReportGeneration reportGeneration;
+    private StringBuilder testReport;
+
+    @Before
+    public void sutUp() {
```

Comment:
```suggestion
    @BeforeClass
    public void setUp() {
```
if you have the same code anywhere else in your project, change it accordingly